src/styles/bloсks/page.scss

```diff
@@ -0,0 +1,22 @@
+.page {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+
+  &__body {
+    margin: 0;
+    padding: 0;
+  }
+
+  &__header {
+    margin: 0 auto;
+  }
+
+  &__catalog {
+    max-width: 944px;
```

Comment:
your width should be calculated depending on how many cards will be in the row, for this you need to create a function that will receive (number of cards * width of these cards) + paddings on the sides.(Checklist pt. 13)

[https://github.com/mate-academy/layout_catalog/blob/master/checklist.md#:~:text=%5BLAYOUT%5D%20%2D%20Use%20sass%20functions%20for%20dynamic%20max%2Dwidth%20calculation%2C%20your%20catalog%20container%20should%20work%20fine%20for%20any%20number%20of%20cards.](url)

Reply:
In my opinion, that does not make sense.